src/foam/lib/formatter/JSONFObjectFormatter.java

```diff
@@ -438,7 +438,7 @@ public void outputJson(String str) {
   }
 
   public void output(ClassInfo info) {
-    outputKey(info.getId());
+    output(info.getId());
```

Comment:
The old code should be correct. Is quoteKeys_ set to true? KeyParser should handle either quoted or unquoted keys. Let me look into the JS parser...